### DISCUSSION TITLE
App name poll (final round)

### DISCUSSION THREAD

**Discussion by naveensingh** [FossifyOrg/General-Discussion#54] [Polls]
The initial and second polls for the app name provided inconclusive results (there was no clear majority). This marks the second round, featuring the most voted two names. The outcome of this poll will be taken into account only if it achieves a minimum of 15% of the votes from the [initial poll](https://github.com/orgs/FossifyOrg/discussions/23) (about 150 votes).

The most-voted name wins.

**Comment by inson1** [comment]
@naveensingh What if there will be less votes?

**Reply by naveensingh** [reply]
@inson1 seems unlikely but in that case, I'll choose it myself instead of doing more polls.

**Reply by inson1** [reply]
@naveensingh I would also set the time. 

Do you know, when there is going to be the release? It has been 2 weeks. And you said 2-3 weeks :) Christmas would be interesting xD if you have time.

**Reply by naveensingh** [reply]
Assuming we push past 150 votes, whatever is winning at the time the apps are ready for release will be considered the winner.

> Christmas would be interesting xD if you have time.

I'm trying to get it done this week but no promises :)

**Reply by DiagonalArg** [reply]
Please make a release now, even if it's just a release of the forked Simple-X apps.

**Reply by dekipantelija** [reply]
Just want to say thank you for staying patient regarding this overblown situation and haooy new yeae! :)

**Comment by gcb** [comment]
fossify is weird as hell in my native language(s), but nothing offensive. and taste is taste so either way is fine as long as it is unique.

"pure" and the other generic names would be too hard to defend from copycats/scammers on the playstore.

**Reply by erebion** [reply]
Yeah, "fossify" sounds just like it is supposed to be a strange sort of verb. But what does it mean to "fossify"?

Sounds a bit like "fossil".

I... don't know. Weird one.

**Reply by kaoneko** [reply]
_Do you Spotify? Then you might as well Fossify too! Here at Fossify we give proprietary software the middle finger: FOSS YOU!_

**Reply by erebion** [reply]
You wouldn't steal a handbag.
You wouldn't steal a television.
You wouldn't download a car.
You wouldn't spotify.

Spotifying is morally a crime and hurts small artists.

**Reply by FoxMoss** [reply]
you're right im going to pirate my music instead

**Reply by erebion** [reply]
Also, I like pure, because it is "purely a calendar app", not just a way to show you ads and gather data, like so many other apps that aren't made for your convenience.

**Reply by Ponytronics** [reply]
I just tipped the hung vote over to Fossify. Oops.

**Comment by SRC267** [comment]
Went with Pure. It feels like a successor to the Simple prefix.

**Reply by gcb** [reply]
pure can be a subtitle. As I mentioned above, there are already tons of spyware ridden apps with ads everywhere, which use those names with pure/simple etc. We shouldn't join those platitudes. Have a proper identifiable name that can copyrighted (if worse came to happen), and use the byline/subtitle/description of pure, immaculate, whatever.

**Reply by gcb** [reply]
Btw, even "simple" which is now bought by an advertising company is an example of what i am talking about :(

**Comment by kumarayush2104** [comment]
Going with Pure, for some reason "fossify" doesn't sound good and also feels like a longer (drag) in terms of name. Also "pure" is more easier to understand, I don't think most people will understand the meaning behind FOSS.

**Reply by SRC267** [reply]
If Pure doesn't become the final name, then that's unlucky for those people who want to seek out an app that keeps things simple (no pun)

**Reply by ikanakova** [reply]
> If Pure doesn't become the final name, then that's unlucky for those people who want to seek out an app that keeps things simple (no pun)

Shakespeare wrote it beautifully: “What's in a name? That which we call a rose, by any other word would smell as sweet.” It doesn't matter if the apps are called Fossify, Pure or otherwise, the important thing is that they carry the original idea of Simple apps "open source Android apps without ads and unnecessary permissions".

**Reply by mosiser** [reply]
Is it really that important that the name should have an easily understandable meaning by everyone?
I don't know anybody that choose a software by the meaning of its name.
I mean... Blender? Gimp? Clementine? Firefox?
It's more important that it is easily distinguishable, that a person could identify an unique name with an unique software, and easily find it.
A generic adjective is used often by scam apps, and nowadays this is the first impression, when I find an app with that name...

**Reply by ikanakova** [reply]
> Is it really that important that the name should have an easily understandable meaning by everyone?
I don't know anybody that choose a software by the meaning of its name.
I mean... Blender? Gimp? Clementine? Firefox?
It's more important that it is easily distinguishable, that a person could identify an unique name with an unique software, and easily find it.
A generic adjective is used often by scam apps, and nowadays this is the first impression, when I find an app with that name...

100% agree with this. Plus, when an app has a specific name, like Gimp, it spreads more easily in the community because you can just recommend Gimp to someone and they'll type it into Google and it won't come up with 1000+ apps like if it was called Simple (Pure) photo editor.

**Comment by Wintandre** [comment]
Every time I recommended the Simple apps to someone, I had to clarify what they should search for because the results would have all kinds of hits from developers including simple in their description etc. Pure is too common a word in English and I can see the same problem occurring again.

Hence I went for Fossify. I might have to spell it out if I'm talking to someone but it should stand out from the crowd.

**Comment by alekksander** [comment]
seems purists are not interested about the name anymore… who would have thought ;)

**Reply by othbert** [reply]
poll for anything 3 times and you're sure to lose people from the funnel, especially as github notifications are whatever

**Reply by dekipantelija** [reply]
Even i only now voted from loss of patience. It feels like an amerocan fillibuster at this point

**Comment by domindel** [comment]
i think both pure and fossify are not good options.
fossify is better than pure, but fossify is also too generic. nothing in the name fossify suggests foss **android** tools.
i think something like fossifydroid could be a better option.

**Reply by DiagonalArg** [reply]
FossyDroid?

**Comment by Neo-Oli** [comment]
Going with fossify, because it implies FOSS in it's name. Pure just sounds like a generic brand name.

**Comment by And96** [comment]
It would be simple if app name = project name.

If the project is Fossify I'll go with Fossify.

Else, I will rename the project too.

**Comment by naveensingh** [comment]
Ok, I'll be updating the app name strings to `Fossify-*`...

I think it's unlikely but as I said before, if this name leads to poor discoverability, I'll consider changing it.

Thanks!

**Comment by Brandonbr1** [comment]
Fossify is a bad app name. Just because you all like Foss does not mean you have to add it to the products app. 

Pure can represent that it is high quality and ad free, as someone said. But most people will not know the meaning of Foss, although most people downloading this are people who know about simple mobile tools messup.

Still its not a vergy good name. Pure or fossify. Pure sound a tad bit generic. Whatever wins, wins I just want this app up.

Honestly. Fresh could be better in my opinion. 

Either way I just want this app on the store to download, because we are at 3 pools already. This has gotten unessscary and confusing.

**Reply by SRC267** [reply]
I think either; Easy, Nice, Fast prefixes will work for.

**Reply by naveensingh** [reply]
> This has gotten unessscary and confusing.

yeah we got carried away :)

**Comment by Alerymin** [comment]
Maybe instead of finding a name prefix try to give a name to the project, name that can be long, and base the prefix on it. For example using the first letter of each word, or combining the words.

**Reply by Ponytronics** [reply]
Like Ccalculator, Cclock and Ggallery?

**Comment by Ponytronics** [comment]
Having followed these three polls and read all the comments, I think that they're a good advertisement for benevolent dictatorships. But, as several people have now suggested, the imperative now is to get the apps out before people forget that Simple apps even existed.

**Comment by ghost** [comment]
@naveensingh I just saw Fossify Gallery hit F-Droid live repos overnight and it's great, thank you!

I just wanted to share that "Pure" is an already crowded word in business, whether it's Pure the UK electronics company, Pure Storage the enterprise appliance folks, Pure Oil (Vitamins), Pure Insurance or... you get the idea. While it might sound neat, it's my opinion to go with a more distinct name than "Pure" to make discoverability of the apps better (Fossify works for example, very unique in search space).

**Reply by dekipantelija** [reply]
Thanks for the heads-up!
